### PR TITLE
Add templateFromAttributes function to browser-utils package

### PR DESCRIPTION
**Pull request by nwalters512**
Lifting this over from PrairieTest, where it's been very useful.

### CODE REVIEW

File: packages/browser-utils/src/template-from-attributes.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    }
+  }
+
+  Object.entries(attributes).forEach(([sourceAttribute, targetSelector]) => {
+    debugLog('---------------------------------------------------------------------');
+    debugLog('templateFromAttributes:');
+    debugLog('source', source);
+    debugLog('target', target);
+    debugLog('sourceAttribute', sourceAttribute);
+    debugLog('targetSelector', targetSelector);
```
Comment (nwalters512) [inline packages/browser-utils/src/template-from-attributes.ts:37]: This is all copied verbatim from PrairieTest. It still hurts me a little bit to see all this, and IMO makes the function unnecessarily hard to read through, though I acknowledge that it's been useful to Matt in the past.

**Review by mwest1066 — APPROVED** [review]

File: packages/browser-utils/src/template-from-attributes.ts (hunk 1 above)
Comment (nwalters512) [reply packages/browser-utils/src/template-from-attributes.ts:37]: I'd be happy with that! I'll add that.

Comment (nwalters512) [reply packages/browser-utils/src/template-from-attributes.ts]: Changed in #e05b1d2.

Comment (mwest1066) [reply packages/browser-utils/src/template-from-attributes.ts]: Nice use of `console.error()`!

**Review by mwest1066 — APPROVED** [review]

**Pull request merged**